NineChronicles.Headless/GraphTypes/States/Models/Item/InventoryType.cs

```diff
@@ -26,15 +28,42 @@ public InventoryType()
                     {
                         Name = "equipped",
                         Description = "filter equipped inventory item"
+                    },
+                    new QueryArgument<ItemSubTypeEnumType>
+                    {
+                        Name = "itemSubType",
+                        Description = "An item subtype for fetching only equipment where " +
+                                      "its subtype is the same. If it wasn't given, you'll " +
+                                      "get all equipment without relationship to the subtype."
+                    },
+                    new QueryArgument<ListGraphType<NonNullGraphType<GuidGraphType>>>
+                    {
+                        Name = "itemIds",
+                        Description = "ItemIds for fetching only equipment where id is in " +
+                                      "the given argument."
                     }),
                 resolve: context =>
                 {
                     var equipments = context.Source.Equipments;
                     var equippedFilter = context.GetArgument<bool?>("equipped");
+                    var itemSubTypeFilter = context.GetArgument<ItemSubType?>("itemSubType");
+                    var itemIdsFilter = context.GetArgument<Guid[]?>("itemIds");
                     if (equippedFilter.HasValue)
                     {
                         equipments = equipments.Where(x => x.equipped == equippedFilter.Value).ToList();
                     }
+
+                    if (itemSubTypeFilter.HasValue)
+                    {
+                        equipments = equipments.Where(equipment => equipment.ItemSubType == itemSubTypeFilter.Value);
+                    }
```

Comment:
```suggestion
                    if (itemSubTypeFilter is not null)
                    {
                        equipments = equipments.Where(equipment => equipment.ItemSubType == itemSubTypeFilter);
                    }
```

Reply:
I applied it ✅ https://github.com/planetarium/NineChronicles.Headless/compare/80f4bb55d4c53832f65b5f2e47c0ec4e3396be02..98754d8e95417044ee290db1599ea8aa32d3961e